server/spec/cards.spec.js

```diff
@@ -0,0 +1,16 @@
+describe("Post request /api/v1/cards", () => {
+  const request = require("request")
```

Comment:
@nikitasakov can't we do `import request from 'request'` at the top of the file?

Reply:
As I understand - no. At least if I write this way, an errors occurs.

Reply:
@nikitasakov when you say that error occurs, please provide it 😉

Reply:
@AleksSenkou 
<img width="689" alt="2018-12-03 18 21 14" src="https://user-images.githubusercontent.com/32716983/49382793-5c0e6580-f728-11e8-9fae-e51819c08403.png">

Reply:
🤔

Reply:
I suppose jasmine knows nothing about babel, so it crashes. Trying to resolve.

Reply:
@datarockets/cuties thoughts about merging with it for now and create an infra task?

@nikitasakov as a PR-holder, please, adjust PR merging as soon as possible so changes can be used for future development 😉

Reply:
@datarockets/cuties looks like it's `request` package responsibility. I mean in [readme](https://github.com/request/request) and in [examples](https://github.com/request/request/issues/3070) aka everywhere the `const request = require("request")` line is used.